.github/workflows/ci.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           'macos_build FluentUITestApp-macOS Debug build test',
           'macos_build FluentUITestApp-macOS Release build test',
           'macos_build FluentUITestApp-macOS Debug build test -destination "platform=macOS,arch=x86_64"',
-          'ios_simulator_build FluentUI-iOS Debug build test -destination "platform=iOS Simulator,name=iPhone 8"', # Provide a destination for the iOS simulator unit tests
+          'ios_simulator_build FluentUI-iOS Debug build',
```

Comment:
I'm realizing now that for FluentUITestApp-macOS and Demo.development, we have one PR check that builds, and then one PR check that builds and then tests. Would we prefer leaving these separate or consolidating to just one check that builds and tests?

Reply:
However, it would make the tests blocking checks, which I'm not sure we want.